pkg/clusteragent/admission/mutate/autoinstrumentation/target_mutator.go

```diff
@@ -56,6 +56,11 @@ func NewTargetMutator(config *Config, wmeta workloadmeta.Component) (*TargetMuta
 		disabledNamespacesMap[ns] = true
 	}
 
+	var tagResolver *TagResolver
+	if config.rcClient != nil {
+		tagResolver = NewTagResolver(config.rcClient)
```

Comment:
Nit - instead of using config, this probably should be passed as an argument to New